src/lib/nvme_pt_ioctl.c

```diff
@@ -1565,11 +1645,12 @@ static int opal_generic_write_table(int fd, struct opal_device *dev,
 		/*
 		 * The append_bytes used below, dependng upon the len either uses
 		 * short_atom_bytes_header (returns 1) or medium_atom_bytes_header
-		 * (returns 2). Hence we consider the MAX of the two i.e, 2.
+		 * (returns 2) or long_atom_bytes_header (returns 4).
+		 * Hence we consider the MAX of the three i.e, 4.
 		 *
 		 * The 1 byte is for the following ENDNAME token.
 		 */
-		uint64_t remaining_buff_size = buf_len - (pos + 2 + 1 + CMD_END_BYTES_NUM);
+		uint64_t remaining_buff_size = buf_len - (pos + 4 + 1 + CMD_END_BYTES_NUM);
```

Comment:
please aovid mixing code with viariable declaration

Reply:
will fix it.